site-content/source/modules/ROOT/pages/development/documentation.adoc

```diff
@@ -18,7 +18,7 @@ instead of visual editors (which some people would call good news).
 Antora processes http://www.asciidoc.org[AsciiDoc], the markup language used to generate our documentation.
 Markup languages allow you to format text using certain syntax elements.
 Your document structure will also have to follow specific conventions.
-Feel free to take a look at https://cassandra.apache.org/doc/latest/[existing documents] to get a better idea how we structure our documents.
+Feel free to take a look at link:/doc/latest/[existing documents] to get a better idea how we structure our documents.
```

Comment:
this changed the wrong link. we want this one to remain as `https://cassandra.apache.org/doc/` (to demonstrate the versions of docs we have available, i.e. "the structure")

it was this [one](url) that needed reverting https://github.com/apache/cassandra-website/pull/157/files#r939723811

Reply:
can you try the following please
```suggestion
Feel free to take a look at link:/doc/[existing documents] to get a better idea how we structure our documents.
```